src/utils/http.ts

```diff
@@ -125,3 +125,4 @@ export const fetch = async (
 }
 
 export const restEndpoints = (resource: string): string => `${rootUrl()}/v2/${resource}`
+export const restBetaEndpoints = (resource: string): string => `${rootUrl()}/v2_beta/${resource}`
```

Comment:
https://github.com/zendesk/sell-vulcand/pull/120:

> Why v2_beta prefix?
Because the API is not yet ready in the Engage service. It is planned to be fully ready for the end of 2020 year.
Also for end of 2020 year we need to integrate Sell Zapier App with this Public API. We want to enable faster testing and integration of our works in Zapier App - thus we will start development using v2_beta APIs. When the APIs will be fully ready we will expose them under serious v2 prefix (without _beta).

Will be moved to **v2/** before releasing Zapier App